backend/apps/matching/migrations/0002_permission_group_creation.py

```diff
@@ -97,6 +99,8 @@ def delete_groups(apps, schema_editor):
         group_list = [
             "is_a",
             "is_b",
+            "approved_a",
+            "approved_b",
```

Comment:
Sorry, late to the party. Why two groups? Wouldn't the combination is_a + approved be enough?

Reply:
yep, good idea